modules/ROOT/pages/streaming-about.adoc

```diff
@@ -122,71 +178,97 @@ In case your use case does not really require the extra memory or performance ov
 
 [IMPORTANT]
 When streaming objects, the in-memory buffer size is measured in instance counts.
 
-When calculating the in-memory buffer size for repeatable auto-paging, you need to estimate how much memory space each instance takes to avoid running out of memory.
+When calculating the in-memory buffer size for repeatable auto-paging, you need
+to estimate the amount of memory each instance takes to avoid running out
+of memory.
 
-As with repeatable streams, you can use different strategies to configure how Mule handles the repeatable auto paging:
+As with repeatable streams, you can use different strategies to configure how
+Mule handles the repeatable auto paging.
 
-=== Repeatable File Store Iterable
+=== Repeatable File Store (Iterable)
 
-This configuration is the default for Mule Runtime Enterprise Edition. +
-It uses a default configured in-memory buffer of 500 objects. If your query returns more results than the buffer size, Mule serializes those objects and writes them to your disk. +
-You can configure the number of objects Mule stores in the in-memory buffer. The more objects you save in memory, the better performance you get by avoiding writing to disk,
+This configuration is the default for Mule Enterprise Edition. This strategy
+uses a default configured in-memory buffer of 500 objects. If your query
+returns more results than the buffer size, Mule serializes those objects and
+writes them to your disk. You can configure the number of objects Mule stores
+in the in-memory buffer. The more objects you save in-memory, the better
+performance you get from avoiding writes to disk.
 
-For example, you can set a buffer size of 100 objects in memory for a query from the SalesForce Connector:
+For example, you can set a buffer size of 100 objects in-memory for a query from
+the SalesForce Connector:
 
+.Repeatable File Store (Iterable):
 [source,xml,linenums]
 ----
 <sfdc:query query="dsql:...">
   <ee:repeatable-file-store-iterable inMemoryObjects="100"/>
 </sfdc:query>
 ----
 
-This interface uses the Kryo framework to serialize objects so it can write them to your disk. +
-Plain old Java serialization fails if the object does not implement the Serializable interface. However if serialization contains another object that doesn’t implement the Serializable interface, Kryo is likely (but not guaranteed) to succeed. For example, a POJO containing an `org.apache.xerces.jaxp.datatype.XMLGregorianCalendarImpl`. Although Kryo serializer allows Mule to serialize objects that the JVM cannot serialize by default, some things can’t be serialized. It's recommended to keep your objects simple.
+Note that MuleSoft uses the Kryo framework for serialization because standard
+Java serialization requires the serialized object (and all its referenced objects)
+to implement the `Serializable` interface. The Kryo serializer, which does not
+have this limitation, is capable of serializing some things that standard Java
```

Comment:
some things > some objects ? "things" is vague

Reply:
done